domains/seyioo.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "For a personnal portfolio.",
+    "repo": "https://github.com/seyiooo/seyiooo.github.io>",
+    "owner": {
+        "username": "seyiooo",
+        "discord": "sey.ioo",
+        "instagram": "sey.ioo"
```

Comment:
You need email line. Though after adding email line you can leave it blank since you have your discord listed